sbin/build.sh

```diff
@@ -224,9 +230,9 @@ runTheOpenJDKConfigureCommandAndUseThePrebuildConfigParams()
 buildOpenJDK()
 {
   if [[ "$OS_MACHINE" == "s390x" ]] || [[ "$OS_MACHINE" == "armv7l" ]] ; then
```

Comment:
I would suggest a documentation in this case and passing the environment setting during build:
`export MAKE_ARGS_FOR_ALL_PLATFORMS="..." makejdk.sh {....}` could be a good start

So for ARM the command should look like 
`export MAKE_ARGS_FOR_ALL_PLATFORMS=" DEBUG_BINARIES=true all" makejdk.sh {....}` could be a good start.

Another option could be to maintain a script that we only use when running outside jenkins which has the checks for the different platform and passes the respective `MAKE_ARGS_FOR_ALL_PLATFORMS` settings.

`MAKE_ARGS_FOR_ALL_PLATFORMS` will also have a default setting of `images`

Which one do you think makes more sense?
I prefer the maintaining the top-level script which would mean it is the doc by itself and would be version controlled. Any other ideas?